sktime/utils/validation/forecasting.py

```diff
@@ -332,17 +331,32 @@ def check_scoring(scoring):
         if object is not an instance of class MetricFunctionWrapper of
         sktime.performance_metrics.
     """
-    from sktime.performance_metrics.forecasting._classes import MetricFunctionWrapper
-    from sktime.performance_metrics.forecasting import sMAPE
+    from sktime.performance_metrics.forecasting._classes import (
+        MetricFunctionWrapper,
+        SquaredMetricFunctionWrapper,
+        PercentageMetricFunctionWrapper,
+        SquaredPercentageMetricFunctionWrapper,
+        AsymmetricMetricFunctionWrapper,
+    )
+    from sktime.performance_metrics.forecasting import MeanAbsolutePercentageError
 
     if scoring is None:
-        return sMAPE()
+        return MeanAbsolutePercentageError(symmetric=True)
 
     if not callable(scoring):
         raise TypeError("`scoring` must be a callable object")
 
-    allowed_base_class = MetricFunctionWrapper
-    if not isinstance(scoring, allowed_base_class):
-        raise TypeError(f"`scoring` must inherit from `{allowed_base_class.__name__}`")
+    allowed_base_classes = (
```

Comment:
Thinking about this, I'm not sure if we need the inheritance check here, in principle users could pass their custom error functions/objects as long as they adhere to the expected API

Reply:
@mloning I was thinking about that when writing the update. Benefit I could see is that it would ensure the user's custom error function adheres to the same `__call__` signature, but I agree it seems like a restrictive way to do that. 

If we wanted, I'm guessing we could check the calIable classes args more directly if we prefer to flag custom classes that don't adhere to the API expectations here rather than when they get called. 

I'll take this check out though when I push my last commit.